README.md

```diff
@@ -36,6 +36,12 @@ jekyll serve
 
 Once you successfully run the moving blog, you can modify the theme and add posts by yourself, have fun!
 
+**Note:** If you want to switch between dark and light themes, navigate to
+
+> assets \> main.scss
```

Comment:
You know, the only place you can put the trigger is `_config.yml`, since many people don't have `assets \> main.scss`. You can refer to https://github.com/huangyz0918/personal-page-blog, how can I change the theme? I don't know where is the `main.scss`.

Reply:
I made the change. Would that do the trick? The trigger is now in confg file, you just need to switch between _sass or _sass-dark in _config.yml